src/patient-registration/field/address/address.component.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import dayjs from 'dayjs';
+import styles from './../field.css';
+
+interface AddressProps {
+  value: {
+    address1: string;
+    address2: string;
+    cityVillage: string;
+    stateProvince: string;
+    postalCode: string;
+    country: string;
+    latitude: string;
+    longitude: string;
+    startDate: Date;
+    endDate: Date;
+  };
+  onChange(address: {
```

Comment:
The interface here seems to be a copy of `value`s type. I would introduce an `AddressValue` interface to avoid copying.

```ts
interface AddressValue {
    address1: string;
    address2: string;
    cityVillage: string;
    stateProvince: string;
    postalCode: string;
    country: string;
    latitude: string;
    longitude: string;
    startDate: Date;
    endDate: Date;
  }

interface AddressProps {
  value: AddressValue;
  onChange(value: AddressValue): void;
}
```